### PR TITLE
add histogram example with binned bin

### PR DESCRIPTION
**Pull request by sirahd**

### CODE REVIEW

**Comment by domoritz** [comment]
Should we delete histogram_bin_transform then?

**Comment by kanitw** [comment]
I think this should replace the existing transform.

File: examples/specs/histogram_binned_transform.vl.json [hunk 1]
```diff
@@ -10,7 +10,16 @@
   "encoding": {
```
Comment (kanitw) [inline examples/specs/histogram_binned_transform.vl.json]: I don't think we should rename the file?  
It's a bin transform, not a binned transform.

**Comment by domoritz** [comment]
Let's squash this.

**Pull request merged**